test/suites/dev/moonbase/test-democracy/test-democracy-referendum.ts

```diff
@@ -101,6 +108,8 @@ describeSuite({
       id: "T03",
       title: "should be votable while staked",
       test: async function () {
+        log(`Disabled test D010904 (Gov V1)`);
```

Comment:
Is there a reason to not just delete these?

Reply:
Yes, we whant to still have the code to have a base to migrate them to gov v2 later

Reply:
Some of the tests can be migrated to OpenGov